core/domain/exp_services_test.py

```diff
@@ -382,15 +380,15 @@ def test_version_number_errors_for_get_multiple_exploration_versions(self):
             feconf.SYSTEM_COMMITTER_ID, self.EXP_ID, change_list, '')
 
         with self.assertRaisesRegexp(
-                ValueError,
-                'Requested version number cannot be higher than the current '
-                'version number.'):
+            ValueError,
+            'Requested version number %s cannot be higher than the current '
+            'version number %s.' % (4, 3)):
```

Comment:
In tests, just hardcode the expected message directly. Don't interpolate constants.

Reply:
Done.